README.md

```diff
@@ -117,13 +117,15 @@ letsencrypt::certonly { 'foo':
 To automatically renew a certificate, you can pass the `manage_cron` parameter.
 You can optionally add a shell command to be run on success using the `cron_success_command` parameter.
 You can optionally add a shell command to be run on before using the `cron_before_command` parameter.
+You can disable emails generated by the cron command using the `disable_cron_emails` parameter.
```

Comment:
should we rephrase this? We don't disable cron emails, we suppress the daemon output.